### PR TITLE
fix(Hybrid Map Listing): fixed ratings in ListingCell for Listing Car…

### PR DESCRIPTION
**Pull request by grenti**
…ousel for the Hybrid Map.

affects: @rentpath/react-ui-core

[Card](https://rentpath.leankit.com/card/607050974)

### CODE REVIEW

File: stories/core/ListingCell.js [hunk 1]
```diff
@@ -32,7 +36,7 @@ export const DefaultListingCell = (
       <div>
         <ListingComponents.Bathroom />
         <h1>hello</h1>
-        <ListingComponents.Ratings />
+        <ListingComponents.Ratings {...ratingProps} />
```
Comment (nray621) [inline stories/core/ListingCell.js]: Why are we changing this to pass props directly to the ratings listing component?
Isn't the point that they just get their props through the `ListingCell`?

Comment (mneumark) [reply stories/core/ListingCell.js]: The idea is to give you the ability to both. So settings like fill color and such can be passed as props. And then the rating score and label will go in via context through `ListingCell`.

Comment (nray621) [reply stories/core/ListingCell.js]: ok cool 👍

**Comment by nray621** [comment]
👍  LGTM

**Comment by bethstevic** [comment]
Looks good 👍

**Pull request merged**